ipsetpy/wrapper.py

```diff
@@ -137,7 +146,10 @@ def ipset_create_set(set_name, type_name, family=None, entry_timeout=None, maxel
     return ipset_send_command(*arguments, command_timeout=command_timeout)
 
 
-def ipset_add_entry(set_name, entry, entry_timeout=None, exist=False, command_timeout=None):
+def ipset_add_entry(set_name, entry, entry_timeout=None, exist=False, command_timeout=None, use_sudo=False):
+    global IPSET_SUDO
```

Comment:
We should pass use_sudo in the same way command_timeout is sent to the ipset_send_command* wrappers.